user-service/index.js

```diff
@@ -17,11 +17,11 @@ app.use(express.urlencoded({ extended: true }));
 app.use(express.json());
 app.use(
   cors({
-    origin: ['http://localhost:3000'],
+    origin: process.env.ENV == 'PROD'? process.env.FRONTEND_URL : 'http://localhost:3000',
     credentials: true,
+    optionsSuccessStatus: 200
   })
 ); // config cors so that front-end can use
-app.options('*', cors());

```

Comment:
Do we not need to enable preflight? We have `UPDATE` and `DELETE` API endpoints though

Reply:
I'm not sure what it does and I thought it overwrites the `cors()` above so I deleted it. I tested `update` and `delete` account on live it seems to work well. Also, I couldn't find the documentation for `.options()` api do you mind sharing if found one?

Reply:
Under Enabling CORS Pre-Flight in https://expressjs.com/en/resources/middleware/cors.html

Reply:
Thanks for sharing, I saw the note below which suggests this.

<img width="701" alt="Screenshot 2022-11-07 at 10 15 57 AM" src="https://user-images.githubusercontent.com/70432640/200212903-67bd8e7d-f9f1-4529-8c41-cc0c412fc6bb.png">

It seems that it doesn't matter anymore once we use `app.use(cors())` but for safety reason, I will just keep it.